packages/next/src/server/lib/revalidate.ts

```diff
@@ -8,17 +8,17 @@ import { CACHE_ONE_YEAR } from '../../lib/constants'
  * value for this option.
  */
 export type Revalidate = number | false
-export type SwrDelta = number
+export type ExpireTime = number
 
 export function formatRevalidate({
   revalidate,
-  swrDelta,
+  expireTime,
 }: {
   revalidate: Revalidate
-  swrDelta?: SwrDelta
+  expireTime?: ExpireTime
 }): string {
-  const swrHeader = swrDelta
-    ? `stale-while-revalidate=${swrDelta}`
+  const swrHeader = expireTime
+    ? `stale-while-revalidate=${expireTime}`
```

Comment:
```suggestion
  const swrHeader = typeof revalidate === 'number' && expireTime !== undefined
    ? revalidate >= expireTime ? '' : `stale-while-revalidate=${expireTime - revalidate}`
```

This number should be the total time from start rather than the delta so we need to subtract `revalidate` from it to get the delta.

If it's the same or less, then it's never stale-while-revalidate.